src/book-club-bans.ts

```diff
@@ -120,7 +137,13 @@ export async function registerBookClubBansListeners(client: Client) {
         await bookClubChannel.send({
           content: `<@${messageSenderId}> has received their ${banCount}${getSuffix(
             banCount
-          )} ban from LGT Book Club, by order of ${getRandomMikeTitle()} Mike. Their crimes against literature continue to stack.`,
+          )} ban from LGT Book Club, by order of ${
+            user.id === MIKE_USER_ID
+              ? getRandomMikeTitle()
+              : getRandomRyanTitle()
+          } ${
+            user.id === MIKE_USER_ID ? ', Mike' : ', Mike'
```

Comment:
The fact that Ryan is included in the list of ryan titles and not in the mike titles leads to this awkwardness (also I'm pretty sure one of these is supposed to be `, Ryan`?)

Refactor the ryan/mike title situation to include Ryan/Mike in the title and remove Mike from here.

Reply:
As an alternative, I'm quite sure there's a way to refactor things to not have a `RYAN_TITLES` array at all. I'm pretty sure you could derive a Ryan title from a Mike title really easily.